HDS Content - Adding Writing Guidelines and Writing Style (#2576)
hashicorp/design-system: website/docs/content/voice-and-tone/index.md

Reviewer: [general question] Does it make sense to break this up how we've done the rest of the content? I know there isn't a lot of content in here right now, but we do plan to add more in the future, so it might be easier long-term to break out the individual markdown files.

Author: removing this for the time being, will add in next iteration.

---
title: Voice and Tone

> caption: How HashiCorp communicates with its users.

Author: Good point, we should probably change this to "our"

> caption: How HashiCorp communicates with its users.

Author: ```suggestion
caption: How HashiCorp communicates with our users.
```

> caption: How HashiCorp communicates with our users.

Reviewer: This and the caption are the same except for one word, should probably one way or the other. Personally, I'd go with:

```suggestion
caption: How HashiCorp communicates with its users.
```

> description: How HashiCorp communicates with its users.

Author: Removed!

previewImage: assets/illustrations/content/tone-and-voice.jpg
---

> “Voice and tone” is how we communicate with our users via copy. The “voice” is our personality and the “tone” is how we express that personality.

Author: We can hold off releasing this this iteration. This can be a swift follow up.

> “Voice and tone” is how we communicate with our users via copy. The “voice” is our personality and the “tone” is how we express that personality.

Author: I deleted the folder because its still rendering. I moved the content to the doc you shared with me.

## Voice

> The HashiCorp voice is smart, professional, technical, to the point, factual, clear, and helpful. Our goal is to be educational and helpful and not portrayed as know-it-alls, mean-spirited, or condescending.

Reviewer 2: ```suggestion
The HashiCorp voice is smart, professional, technical, to the point, factual, clear, and helpful. Our goal is to be educational and helpful and not come across as know-it-alls, mean-spirited, or condescending.
```

Reviewer 2: could also replace "portrayed" with "portray ourselves" as another option.

## Tone

> We're guiding our customers on a transformational journey to the cloud as educational stewards to illuminate their path to success. Our message should be smart, helpful, and to the point to enable human conversations.

Reviewer 3: "To the point" and "helpful" are repeated here (helpful three times) which is slightly odd. Are there other words we could use there that make it less repetitive?

### Smart

> We are professional and our conversations are pragmatic and kind.

Reviewer: [suggestion] Given how the other content following this sentence is written, would make more sense to match the formatting?

```suggestion
Our conversations are professional, pragmatic, and kind.
```
That makes it a bit shorter, but I'll leave it up to you if it makes sense.
